### PR TITLE
Add M3: Interactive List functionalities

### PR DESCRIPTION
**Pull request by lmunozm1702**
### This deliverable, includes:

- Badge with active task total counter.
- Multiple task remove (all marked as completed in checkbox), by "Clear al completed link".
- Event listener in checkbox to mark as completed/active a task.
- Store and update task in localStorage.

Update README.md with total scope of the project.

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -195,3 +203,25 @@ h1 {
...
+  font-size: 9px;
+  font-weight: 400;
+  line-height: 1;
+  color: #fff;
+  text-align: center;
+  white-space: nowrap;
+  vertical-align: baseline;
+  background-color: red;
+  border-radius: 10px;
+}
```
Comment (codecaiine) [inline src/style.css:228]: - [x] Good job so far your project is impressive. It works as expected :100: But I would like to recommend you reduce the `width` of your todo list website displays a list that looks and behaves like the part of the minimalist project captured in the video below :arrow_down: 

#### Capture design
68747470733a2f2f696d672e796f75747562652e636f6d2f76692f416355642d5f596a6a71672f302e6a7067 (1)

#### Your design
Screenshot from 2022-11-02 20-55-41

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @lmunozm1702 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- App works as expected :heavy_check_mark: 
- No linters errors :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style.css (hunk 1 above)
Comment (lmunozm1702) [reply src/style.css:228]: Hi! @codecaiine i set the max-with in 490 px as the real site in https://web.archive.org/web/20180320194056/http://www.getminimalist.com:80/.
Regards!

image

**Review by KDlamini — APPROVED** [review]
### Hi @lmunozm1702 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 

✔️  Good PR title and description
✔️  UI follows the given design
✔️  No linter errors

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**